seatunnel-dist/src/main/assembly/assembly-bin.xml

```diff
@@ -52,7 +52,12 @@
         <!-- ============ Starter Bin ============  -->
         <!--connector starter v2-->
         <fileSet>
-            <directory>../seatunnel-core/seatunnel-flink-starter/src/main/bin</directory>
+            <directory>../seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/bin</directory>
```

Comment:
cascade update docs `start-seatunnel-flink-connector-v2.sh`？

https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/start-v2/locally/quick-start-flink.md
https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/command/usage.mdx

Reply:
Document update I will submit another pull request to reduce the pressure of review. What do you think about it? Because this can cause lots of update.

Reply:
👌